app/templates/people/person/personal-information/index.hbs

```diff
@@ -1,224 +1,93 @@
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>Telefoonnummer</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>{{@model.mandatories.firstObject.contacts.firstObject.telephone}}</p>
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div class="au-o-grid">
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>Adres</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>{{@model.mandatories.firstObject.contacts.firstObject.contactAddress.fullAddress}}
-                      </p>
-                    </div>
-                  </div>
-                </div>
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>GSM nummer</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p></p>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </c.content>
-        </AuCard>
-      </div>
-    </div>
+    <DataCard>
+      <:title>Contactgegevens:
+        {{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.classification.label}}
+      </:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Organisatie</:label>
+            <:content>
+              <AuLink
+                @model={{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.id}}
+                @route="administrative-units.administrative-unit">
+                {{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.name}}
+              </AuLink>
+            </:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
+            <:label>Email</:label>
+            <:content>{{@model.person.mandatories.firstObject.contacts.firstObject.email}}</:content>
+          </Item>
+          <Item>
+            <:label>Telefoonnummer</:label>
+            <:content>{{@model.person.mandatories.firstObject.contacts.firstObject.telephone}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
+            <:label>Adres</:label>
+            <:content>{{@model.person.mandatories.firstObject.contacts.firstObject.contactAddress.fullAddress}}</:content>
+          </Item>
+          <Item>
+            <:label>GSM nummer</:label>
```

Comment:
Since we don't have data for this yet I would hide it for now maybe?

Reply:
Ok :)

Reply:
Yes, we have this!

Reply:
Sorry, I thought it was the address!